crates/noirc_evaluator/src/ssa_refactor/ir/function.rs

```diff
@@ -80,6 +95,10 @@ impl Function {
         }
         blocks
     }
+
+    pub(crate) fn compile_to_brillig(&mut self) {
+        self.dfg.brillig_gen(BrilligGen::compile());
+    }
```

Comment:
This looks like it should be a method on `BrilligGen` instead:
```rs
impl BrilligGen {
    fn compile_function(&mut self, function: &Function) -> Brillig {
        todo!()
    }
}
```

Reply:
It is already a BrilligGen method, but then we need to set the compilation result into the dfg.

Reply:
I think Brillig should be separate from the Ssa like Acir is, rather than being contained inside of it

Reply:
Agreed, its nice to have the separation between Acir and SSA IR because then we can have more modular code

Reply:
Done